crates/oxc_ast/src/visit.rs

```diff
@@ -1064,7 +1064,7 @@ pub trait Visit<'a>: Sized {
             }
             JSXAttributeValue::Element(elem) => self.visit_jsx_element(elem),
             JSXAttributeValue::Fragment(elem) => self.visit_jsx_fragment(elem),
-            JSXAttributeValue::StringLiteral(_) => {}
+            JSXAttributeValue::StringLiteral(lit) => self.visit_string_literal(lit),
```

Comment:
oh - before we didn't check jsx string literals, that's why its failing now.

`<foo attr='\d'></foo>` this feels like a case that should be in the fail list anyway?

Reply:
https://facebook.github.io/jsx/#sec-jsx-string-characters

1.5.1 JSX String Characters
Historically, string characters within [JSXAttributeValue](https://facebook.github.io/jsx/#prod-JSXAttributeValue) and [JSXText](https://facebook.github.io/jsx/#prod-JSXText) are extended to allow the presence of [HTML character references](https://facebook.github.io/jsx/#sec-HTMLCharacterReference) to make copy-pasting between HTML and JSX easier, at the cost of not supporting \ [EscapeSequence](https://tc39.es/ecma262/#prod-annexB-EscapeSequence) of ECMAScript's [StringLiteral](https://tc39.es/ecma262/#prod-StringLiteral). We may revisit this decision in the future.

hmmm ... it's a valid test case. Let's create an issue for this.

Reply:
https://github.com/web-infra-dev/oxc/issues/1061